Orange/widgets/visualize/owvenndiagram.py

```diff
@@ -218,6 +218,8 @@ def itemsetAttr(self, key):
         attr_index = combo.currentIndex()
         if attr_index >= 0:
             return model[attr_index]
+        elif combo.count() is 0:
+            return -1;
```

Comment:
Please use `==` operator to compare integers for equality. The fact that small integers in cpython are singletons is an implementation detail that should not be relied upon.